google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/SelectConfigDestinationFolderDialog.java

```diff
@@ -53,6 +55,25 @@ public SelectConfigDestinationFolderDialog(@Nullable Project project) {
         project,
         FileChooserDescriptorFactory.createSingleFolderDescriptor()
     );
+
+    if (project != null && project.getBasePath() != null) {
+      String srcPath = project.getBasePath() + "/src";
+      String srcMainPath = project.getBasePath() + "/src/main";
+      String AppenginePath = project.getBasePath() + "/src/main/appengine";
+      String dockerPath = project.getBasePath() + "/src/main/docker";
+
+      if (fileType == ConfigFileType.APP_YAML && new File(AppenginePath).isDirectory()) {
```

Comment:
I wonder if it makes sense to default it to the canonical directories even if they don't already exist? In this case it should prompt the user to create them (I think this feature was added previously).